src/ctia/entity/feed.clj

```diff
@@ -139,7 +140,7 @@
           (fn [feeds]
             (map #(decrypt-feed % services) feeds))))
 
-(def fetch-limit 200)
+(def fetch-limit 10000)
```

Comment:
The ElasticSearch documentation recommends to not request too many results at once with `from` and `size`. It can significantly increase memory and CPU usage. Why did you change this limit in the context of this PR?